cognite/client/_api/data_modeling/views.py

```diff
@@ -239,6 +239,65 @@ def apply(self, view: ViewApply | Sequence[ViewApply]) -> View | ViewList:
                 ...    )
                 ... ]
                 >>> res = c.data_modeling.views.apply(views)
+
+
+            Create views with edge relations::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes.data_modeling import (
+                ...     ContainerId,
+                ...     DirectRelationReference,
+                ...     MappedPropertyApply,
+                ...     ViewApply,
+                ...     ViewId,
+                >>> )
+                >>> from cognite.client.data_classes.data_modeling.views import (
+                ...     MultiEdgeConnectionApply,
+                >>> )
+                >>> c = CogniteClient()
+                >>> movie_view = ViewApply(
+                ...     space="imdb",
+                ...     external_id="Movie",
+                ...     version="1",
+                ...     name="Movie",
+                ...     properties={
+                ...         "name": MappedPropertyApply(
+                ...             container=ContainerId("imdb", "Movie"),
+                ...             name="name",
+                ...             container_property_identifier="name",
+                ...         ),
+                ...         "actors": MultiEdgeConnectionApply(
+                ...             type=DirectRelationReference(
+                ...                 space="imdb", external_id="Movie.actors"
+                ...             ),
+                ...             source=ViewId("imdb", "Actor", "1"),
+                ...             name="actors",
+                ...             direction="outwards",
+                ...         ),
+                ...     },
+                >>> )
```

Comment:
```suggestion
                ... )
```

Reply:
stupid question but is there a tool that does this formatting?